tests/functional/spec/features/section_summary/section_summary.spec.js

```diff
@@ -24,6 +29,12 @@ describe("Section Summary", () => {
       expect($(PropertyDetailsSummaryPage.submit()).getText()).to.contain("Continue");
     });
 
+    it("When I get to the section summary page, Then it should have the correct table headers set'", () => {
+      expect($(PropertyDetailsSummaryPage.propertyDetailsSummaryTableHead()).getHTML()).to.contain("Question");
+      expect($(PropertyDetailsSummaryPage.propertyDetailsSummaryTableHead()).getHTML()).to.contain("Answer given");
+      expect($(PropertyDetailsSummaryPage.propertyDetailsSummaryTableHead()).getHTML()).to.contain("Change answer");
```

Comment:
These shouldn't be functional tests, you are checking the source HTML. These should be in integration tests in which you can test the HTML.